documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -29,6 +29,15 @@ default name resolve correctly.
 This setting has no default and must always be provided.
 |===
 
+.Optional parameters
+
+|===
+|Parameters |Default |Description
+|`provisioningDHCPExternal`| |Defines if external DHCP will be used or the one configured by installer
+|`provisioningDHCPRange`| |   i.e. 172.22.0.10,172.22.0.100  Defines the IP range to use for hosts on the provisioning network.
+|`provisioningNetworkCIDR`| | <CIDR> # i.e. fd00:1101::1/64
+|===
+
```

Comment:
While we are at it, lets go ahead and add additional optional parameters. I'm going to include them here so you can include in your PR. I went ahead and did some column clean up too for easier readability. I hope it renders properly :D 

```
[cols="1,1,3", options="header"] 
.Optional Parameters
|===
|Parameters
|Default
|Description

|`provisioningDHCPExternal`
| false
|Defines if external DHCP will be used or the one configured by installer

|`provisioningDHCPRange`
| `172.22.0.10,172.22.0.100`
|Defines the IP range to use for hosts on the provisioning network.

|`provisioningNetworkCIDR`
| `172.22.0.0/24`
| The CIDR for the network to use for provisioning.

| `clusterProvisioningIP`
| `172.22.0.3`
| The IP within the cluster where the provisioning services run.

| `bootstrapProvisioningIP`
| `172.22.0.2`
| The IP on the bootstrap VM where the provisioning services run while the 
control plane is being deployed.

| `externalBridge`
| `baremetal`
| The name of the bridge of the hypervisor attached to the external network.

| `provisioningBridge`
| `provisioning`
| The name of the bridge on the hypervisor attached to the provisioning network.

| `defaultMachinePlatform`
| 
| The default configuration used for machine pools without a platform 
configuration.

| `bootstrapOSImage`
| 
| A URL to override the default operating system image for the bootstrap node. 
The URL must contain a sha256 hash of the image. 
Example https://mirror.openshift.com/rhcos-<version>-qemu.qcow2.gz?sha256=abc...

| `clusterOSImage`
|
| A URL to override the default operating system for cluster nodes. 
The URL must include a sha256 hash of the image. 
Example https://mirror.openshift.com/images/rhcos-<version>-openstack.qcow2.gz?sha256=abc...

|===
```

Reply:
/lgtm

Reply:
We will also need to filter this list based on release

4.4 and 4.5 have all of the above from the list.

Version 4.3 **does not** have:
```
ProvisioningNetworkCIDR
ProvisioningDHCPExternal
ProvisioningDHCPRange
```

I got the list from https://github.com/openshift/installer/blob/release-4.5/pkg/types/baremetal/defaults/platform.go , feel free to change release to get the details you need.

Reply:
Done, with conditional for > 4.3 to show the extra parameters